From 08723f30072fcfcee2d061271bec7d79668dc869 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Fri, 12 May 2017 14:37:55 +0200
Subject: [PATCH xkeyboard-config] Map evdev keycode KEY_SOUND to
 XF86AudioPreset

KEY_SOUND is used on actual devices and recent xproto versions
defines the AudioPreset keysym for this key.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 symbols/inet | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/symbols/inet b/symbols/inet
index ae1f6ee..b97ccf2 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -184,7 +184,7 @@ xkb_symbols "evdev" {
     key <I218>   {      [ Print                 ]       };
 //  key <I219>   {      [ ]       }; // KEY_HP
     key <I220>   {      [ XF86WebCam            ]       };
-//  key <I221>   {      [ ]       }; // KEY_SOUND
+    key <I221>   {      [ XF86AudioPreset       ]       };
 //  key <I222>   {      [ ]       }; // KEY_QUESTION
     key <I223>   {      [ XF86Mail              ]       };
     key <I224>   {      [ XF86Messenger         ]       }; // KEY_CHAT
-- 
2.21.0

